src/lib/api-client.js

```diff
@@ -184,3 +202,14 @@ export function getModelsForNewItem() {
         hashHistory.push('/items/new');
     });
 }
+export function deleteItem(address){
+    return del('item', {
+        itemAddress: address
+    }).then(data => {
+        Dispatcher.handleAction('ITEMS_RECEIVED', data);
```

Comment:
Should this be an ITEM_DELTED action, so that we can more easily add a toast for it? We can have the ItemStore also listen to ITEM_DELETED so it has the same effect as ITEMS_RECEIVED currently does.